### ISSUE TITLE
Duotone filters are missing from live site when used in Classic Themes

### ISSUE THREAD

**Issue by cuemarie** [WordPress/gutenberg#54099]
### Description

On sites using a classic theme, Duotone filters apply in the post editor, but don't appear on the live site

### Step-by-step reproduction instructions

1. Activate a classic theme such as Twenty Twenty One
2. Open a new post and add an image block
3. Add a duotone filter to the image 
     - _(observe that this filter appears in the editor)_
4. Save and view the post
     - _(observe that the filter does not appear on the live site)_

### Screenshots, screen recording, code snippet

https://github.com/WordPress/gutenberg/assets/27249804/ea71cf8c-e2ef-4647-a0fe-6d10f2744d9e

Inspecting the image block shows `wp-duotone-rgb10400-rgb255232232-1`, even though the filter is not applied.

![Markup on 2023-08-31 at 15:16:20](https://github.com/WordPress/gutenberg/assets/27249804/b9547e11-5637-4903-91f0-b3c5b88de773)

### Environment info

- WP 6.3.1
- GB 16.5.1

Deactivating Gutenberg altogether allows the Duotone filter to appear on the live site.

Possibly related? https://github.com/WordPress/gutenberg/issues/49293 (I can't tell if this is the same issue - it appears to just be about the default colors available, and not the function of duotone on the live site).

Many WordPress.com customers have reported this issue; those reports are being tracked here: https://github.com/Automattic/wp-calypso/issues/78442

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by bph** [comment]
I can reproduce this with the theme Twenty-Twenty-Thirteen for WordPress nightly and Gutenberg nightly. 
The Duotones selected don't show on the front- end. 

https://github.com/WordPress/gutenberg/assets/39980/c2a0856f-cc16-4534-9c66-22fb9444ea86

**Comment by bph** [comment]
Tested it with WordPress 6.2.2 (no GB plugin) and the duotone works for both, Editor and front-end. 

Then I was curious, which Gutenberg plugin version broke the duotone in classic themes. 
Gutenberg 15.6.2 it still worked
Gutenberg 15.7.0 it stopped working. 

The regression might have been introduced via this PR 

- #49706

@ajlende when you get a quiet moment, could you take a look at this?

**Comment by t-hamano** [comment]
This issue was also reproducible in WordPress 6.3 when I activated the latest Gutenberg plugin.

- ✅ The image block has a class called `wp-duotone-XXXXXX-XXXXXXXXXXX-X` in the block wrapper element.
- ✅ A filter element with an ID attribute identical to its class name is also rendered.
- ❌ However, the inline style for applying that filter is not output.

**Comment by nderambure** [comment]
Any update on this ?
Broken Duotone on classic themes is still there in WordPress 6.4.
I have some websites using intensively duotone to work nicely with the design of the custom theme and my eyes are burning :)

**Comment by t-hamano** [comment]
Hi @nderambure,

This issue should have been fixed in WP Core:

[#58734 (Duotone filter not reflected on the frontend) – WordPress Trac](https://core.trac.wordpress.org/ticket/58734)

However, the code modified in the core is not backported to the Gutenberg plugin. Therefore, if you have the Gutenberg plugin enabled, I think this issue will reoccur.

I have submitted #54778 to also fix the issue with the Gutenberg plugin.
